Update OIDC var description in README (#2086)
artefactual/archivematica: src/archivematica/dashboard/install/README.md

- **`OIDC_ACCESS_ATTRIBUTE_MAP`**
  - **Description:** Set OIDC token details to extract. This string should be

>     JSON-decodable.  If `OIDC_OP_SET_ROLES_FROM_CLAIMS` is set to `True` then
>     `"realm_access": "realm_access"` must be added to the default config for
>     this setting.

Reviewer: Nitpick: I find the "default" part of the new sentence a bit hard to understand. When users customize and provide their own setting value it's not the **default** anymore, right? What about something along the lines of:

```suggestion
    JSON-decodable.  If `OIDC_OP_SET_ROLES_FROM_CLAIMS` is set to `True` then
    the entry `"realm_access": "realm_access"` must be included in this setting.
```
